channels/urls.py

```diff
@@ -20,6 +22,16 @@
         name='post-list',
     ),
     url(
+        r'^api/v0/channels/(?P<channel_name>[A-Za-z0-9_]+)/contributors/$',
+        ContributorListView.as_view(),
+        name='contributors-list',
```

Comment:
Can you change this to `contributor-list` and the below to `contributor-detail`, for consistency with the other URL names?